test/unit/Site.test.js

```diff
@@ -191,14 +191,14 @@ test('Site read site config for custom site config', async () => {
 
 test('Site resolves variables referencing other variables', async () => {
   const json = {
-    'lib/asset/glyphicons.csv': '',
+    'lib/asset/glyphicons.csv': 'glyphicon-plus',
     'lib/template/page.ejs': PAGE_EJS,
     'site.json': SITE_JSON_DEFAULT,
     '_markbind/variables.md':
     '<span id="level1">variable</span>'
     + '<span id="level2">{{level1}}</span>'
-    + '<span id="level3">{{level2}}</span>'
-    + '<span id="level4">{{level3}}</span>',
+    + '<span id="level3">{{glyphicon_plus}}</span>'
+    + '<span id="level4">{{level3 | safe}}</span>',
```

Comment:
I don't really like this workaround, but without doing so, nunjucks will escape the angular brackets and the glyphicon code will be render literally:

---

`variables.md`:
```html
<span id="icon_prereq">{{glyphicon_education}}</span>
<span id="icon_ref_prereq_nosafe">{{icon_prereq}}</span>
<span id="icon_ref_prereq_safe">{{icon_prereq | safe}}</span>
```

`index.md`
```
{{icon_prereq}}

This variable didn't use 'safe' when referencing: {{icon_ref_prereq_nosafe}}.

This variable did: {{icon_ref_prereq_safe}}.
```

Results:

![230-safe](https://user-images.githubusercontent.com/3168908/40819038-d408e21e-658c-11e8-8ca8-163f162dd161.png)

Reply:
Disabling HTML escape globally (which is the same effect of applying "safe" to all variables) doesn't seem to be a good alternative either.

Reply:
Perhaps we should document this. 

I was also thinking some people may want to reference glyphicons in other variables and maybe concatenate them like

`<span id="icon_alert_prereq">{{ icon_alert | safe}}{{icon_prereq | safe}}</span>`

This will be fine right?

Reply:
Noted, will document about the flaw.

> ```
> <span id="icon_alert_prereq">{{icon_alert | safe}}{{icon_prereq | safe}}</span>
> ```
>
> This will be fine right?

Yep, that works.